app/pods/cases/case/subcases/overview/controller.js

```diff
@@ -13,6 +15,24 @@ export default class CasesCaseSubcasesOverviewController extends Controller {
 
   @action
   refreshSubcases() {
-    this.send('refreshParentModel');
+    this.send('refreshSubcasesRoute');
+  }
```

Comment:
why send when we can router with the polyfill? :D

Reply:
hah, good catch!